### PR TITLE
Use template in app markup

### LINKED ISSUE #86: Use template on app markup
## Problem

When building the test app we are using duplicate markup which exists in `src/web/praxisIsncsciAppLayout/appLayoutTemplate.ts`.

We should use the template markup straight into index.html. This should be automated during the build process.

### PR DESCRIPTION
**Pull request by EddieMachete**
Closes #86 

## Problem

When building the test app we are using duplicate markup which exists in `src/web/praxisIsncsciAppLayout/appLayoutTemplate.ts`.

We should use the template markup straight into index.html. This should be automated during the build process.

## Solution

We are adding a `postbuild` step that will replace the constant `{{app-template}}` in the `build/index.html` with the template in `src/web/praxisIsncsciAppLayout/appLayoutTemplate.ts`. For the template to work in our build pipeline, we had to add a config to `Rollup` so we can export the template function as `cjs` and use it in **node**.

## Testing

- [x] 1. Can inject the template markup into `index.html`

<details>
  <summary>Test case</summary>

  1. Navigate to `assets/index.html`
  2. Confirm the string `{{app-template}}` is present
  3. Execute `npm run build`
  4. Open `build/index.html`
  5. Confirm the string `{{app-template}}` was replaced with the template markup

</details>

### CODE REVIEW

File: assets/index.html
Comment (EddieMachete) [inline assets/index.html:1]: We get to clean this file and reuse the markup in the template

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "url": "git+https://github.com/praxis-isncsci/ui.git"
   },
   "scripts": {
+    "postbuild": "node ./scripts/update-template-markup.ts",
```
Comment (EddieMachete) [inline package.json:73]: Here we add the build step that will do the string replacement

File: rollup.config.mjs [hunk 2]
```diff
@@ -45,6 +45,17 @@ const demoSiteConfig = {
   ],
 };
 
+const templateConfig = {
```
Comment (EddieMachete) [inline rollup.config.mjs:48]: I added this step to rollup so we can generate a `cjs` module which we can then use during our build process

**Review by praxis-tech — APPROVED** [review]
Ship it!

**Pull request merged**